lib/plugins/aws/provider/awsProvider.js

```diff
@@ -498,6 +498,25 @@ class AwsProvider {
     return `${provider.deploymentPrefix}`;
   }
 
+  getAlbTargetGroupPrefix() {
+    const provider = this.serverless.service.provider;
+    if (
+      provider.alb === null ||
+      provider.alb === undefined ||
+      provider.alb.targetGroupPrefix === null ||
+      provider.alb.targetGroupPrefix === undefined
+    ) {
```

Comment:
Let's simplify condition to `if (!provider.alb || !provider.alb.targetGroupPrefix)`

Reply:
Suggestion adopted